config/schemas/mod.json

```
@@ -56,6 +56,13 @@
 			"description": "List of mods that can't be enabled in the same time as this one",
 			"items": { "type":"string" }
 		},
+		"compatibility" : {
+			"type":"object",
+			"description": "List of supported version of vcmi engine",
+			"additionalProperties" : {
+				"type" : "string"
+			}
```

My fault for not making schemas for this
```suggestion
			"description": "Supported versions of vcmi engine",
			"additionalProperties" : false,
			"properties" : {
				"min" : {
					"type" : "string",
					"description" : "minimal compatible vcmi engine version in a format major.minor.patch. When specified, earlier versions won't be supported",
					"pattern" : "^\d+\.\d+\.\d+$"
				},
				"max" : {
					"type" : "string",
					"description" : "maximum compatible vcmi engine version in a format major.minor.patch. When specified, later versions won't re supported",
					"pattern" : "^\d+\.\d+\.\d+$"
				}
			}
```

---

Done (partially). Looks like our schema validation does not supports checks by pattern, so for now game will only check for presence of min/max fields